### PR TITLE
Push image with 'latest' tag in CI

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

Fixes: #52 

This PR goes after #63

### CODE REVIEW

**Comment by ashish-amarnath** [comment]
hold for #63

File: hack/docker-push.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Copyright 2019 the Velero contributors.
```
Comment (carlisia) [inline hack/docker-push.sh:3]: s/2019/2020 :)

Comment (ashish-amarnath) [reply hack/docker-push.sh]: beat me to it! fixed now :)

File: hack/docker-push.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```
Comment (ashish-amarnath) [inline hack/docker-push.sh:1]: mostly copied from https://github.com/vmware-tanzu/velero/blob/master/hack/docker-push.sh

**Comment by carlisia** [comment]
@nrb is this still pending anything?

**Pull request merged**